salt/addons/dex/manifests/15-configmap.yaml

```diff
@@ -43,6 +43,36 @@ data:
           groupAttr: uniqueMember
 
           nameAttr: cn
+    {% for con in pillar['dex_connectors'] %}
```

Comment:
If we swap this out for something like this:

    {{ salt['pillar.get']('dex_connectors', []) }}

I believe this will then pass CI, and avoid blocking this PR on the Velum side of this change.

As a side note - can we "namespace" the keys? We've been slowly migrating towards a pattern that looks like `dex:connectors` in salt, and `{"dex": "connectors": [...]}` in JSON.

Reply:
Agreed that adding the default value of empty array would be better and allow the change to go in without breaking prior to the output of the connectors. I will make that change and update this PR.

As for pulling connectors under a dex key, that change could certainly be made, but I am not sure what benefit it would have as there is no other dex configuration output besides the connectors coming out of the pillars.

Reply:
> As for pulling connectors under a dex key, that change could certainly be made, but I am not sure what benefit it would have as there is no other dex configuration output besides the connectors coming out of the pillars.

We only have connector configuration, for now... Down the line, we will likely end up with more options - having them all nested together under a `dex` key will make this cleaner.

Reply:
Understood; I have updated both PRs to use a dex key to group configuration. Offhand it also does make sense as we will need information related to groups there in the future.